### ISSUE TITLE
[DOCS] <Minor improvisations in the contents of the home page>

### ISSUE THREAD

**Issue by SargamPuram** [anmode/grabtern-frontend#196]
### Description

Hey , 
I found some headings in the home page that could be improvised . The changes would make it more grammatically correct and make it sound better.
Kindly review the issue and if you find it okay , please assign it to me as a part of GSSOC 2023

### Screenshots

1. The 'What Grabtern Do for their Students' can simply be changed to 'What Grabtern does for it's students'
![Grabtern1](https://github.com/anmode/grabtern-frontend/assets/130867468/fe568a8e-c0d2-456e-8d76-5e4b9523f6aa)
2. The 'Why to be Mentor at Grabtern' can be replaced by 'Why Mentor At Grabtern'
![Grabtern2](https://github.com/anmode/grabtern-frontend/assets/130867468/22fcffc7-42f2-4742-a98b-3f0cb25585a4)

**Comment by SargamPuram** [comment]
Kindly review the issue and if you find it okay , please assign it to me as a part of GSSOC 2023

**Comment by anmode** [comment]
Are you up for redesigning or improving the UI of Home page...
Ideation - https://mentro.tech/ ( Animations)
https://unstop.com/mentor for more.   It would make it as level 2

**Comment by SargamPuram** [comment]
@anmode , SURE , I'm up for it . Kindly give me more details . Like whether or not the animations will be provided to me from your end or not . And also if you want a complete redesign of the current UI or just some minor enhancements based on the websites you pinned as Ideation . 
Also if you want I could finish this issue and create a separate issue for the tasks mentioned by you and work on it . Your call .

**Comment by SargamPuram** [comment]
> If you are up for complete changes in design..that is ok. We can start with Figma design. could you give me some time. I'll confirm you tomorrow. Thank you
@anmode 
Yeah , Sounds good . Figma works for me . No worries , Take your time.

**Comment by SargamPuram** [comment]
@anmode 
Till then if it's okay with you , you could assign me the above issue I'll work on it by that time

**Comment by anmode** [comment]
Sure!

**Comment by SargamPuram** [comment]
@anmode I have linked a PR that will close this issue . Can you please review it . And If you find it okay please add labels and merge it

**Comment by anmode** [comment]
@SargamPuram We got guidelines that one line code or two line code... will not get any levels. And I disscussed that `Why to be mentor at grabtern` is ok...yeah but first one yours ok!
Would you like to work on nav bar? i just created issue it would be nice contribution

**Comment by SargamPuram** [comment]
> @SargamPuram We got guidelines that one line code or two line code... will not get any levels. And I disscussed that `Why to be mentor at grabtern` is ok...yeah but first one yours ok! Would you like to work on nav bar? i just created issue it would be nice contribution

@anmode that's completely understandable . Even I wish to adhere to the community guidelines . Just let me know then should I just close this issue or what should I do next .
Also would love to work on the responsiveness of the nav bar but just checked that it has already been assigned to someone . 
Been thinking of adding a Student Testimonial section or page . OR overall improving the UI of the page . Just let me know your thoughts about it .

**Comment by SargamPuram** [comment]
@anmode also if nothing else is possible . I am ok that the current PR gets merged without levels , just the gssoc , enhancements  is fine too . I can change it back to Why to be mentor at grabtern ? if that's what you are looking for
I'll work on making some more significant contributions in the future so that they'll pass the level check .

**Comment by anmode** [comment]
it is ok! Please revert to the second statement you changed. I'll merge it. and further, you wanna work. if you are familier with GitHub automation you can create bots.

Or you can create a login card. I hope no one is working on that. login card is like go to gmail.. when you click on profile a card comes... so create a card component so it can be used anywhere.

**Comment by SargamPuram** [comment]
@anmode 
Just reverted it back !
Can you tell me the usecase of the bot you are looking for . Do you want the usual one which displays 'Thanks for raising the issue ' on the issue thread , and comments on the PR requests , mergers , etc. 
Also about the login card I think I get what you are saying , but can you elaborate where else can the card component be used . and the contents on the card has to be just login , my profile ?

**Comment by SargamPuram** [comment]
Also , thank you for being so supportive . And suggesting me various issues where I can contribute .

**Comment by SargamPuram** [comment]
![Reference](https://github.com/anmode/grabtern-frontend/assets/130867468/b72fdb28-717f-4d34-b9a6-94534f19ab17)
Is this what you are talking about . Similar to udemy.

**Comment by anmode** [comment]
> @anmode 
> Just reverted it back !
> Can you tell me the usecase of the bot you are looking for . Do you want the usual one which displays 'Thanks for raising the issue ' on the issue thread , and comments on the PR requests , mergers , etc. 
> Also about the login card I think I get what you are saying , but can you elaborate where else can the card component be used . and the contents on the card has to be just login , my profile ?

Tomorrow I'll attach UI. How card looks like.
Nd for bot yess.
A fully autmoted bot for greeting for first time contributor..
There are numerous cases and there greeting them differently.
1. First time.
2. Not new
3. Collaborator creating issue
4. Author/owner creating issue.

Pr bot differently..
You can start by simply greet bot them we will move to different cases. Thank you

**Comment by SargamPuram** [comment]
Okay , then looking forward for the UI design .
Till then I'll to create a simple greet bot as you've mentioned since I've had only limited experience of github automation . But will still try and create a useful bot.

**Comment by AmanJain18** [comment]
@SargamPuram Thanks for ur patience and appreciation ✌🏻❤️. Will close these PR Now, and if u have any idea or suggestions, huat ping us at discord.
